### PR TITLE
Add a emoji to highlight Travis status

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

**Comment by ghost** [comment]
Tests SUCCESS for HEAD 864c0cd5baa82b5abe650be8fd33ab49806dc655

- [TOXENV=pep8](https://api.travis-ci.org/jobs/315514710/log): PASSED
- [TOXENV=py27](https://api.travis-ci.org/jobs/315514711/log): PASSED
- [TOXENV=py35](https://api.travis-ci.org/jobs/315514712/log): PASSED

File: pastamaker/gh_pr.py [hunk 1]
```diff
@@ -88,10 +88,17 @@ def pastamaker_travis_post_build_results(self):
         self.head.sha)]
     for i, job in enumerate(self.pastamaker["travis_detail"]["jobs"]):
         try:
-            message.append('- [%s](%s): %s' % (
+            state = job["state"].upper()
+            if state == "PASSED/":
```
Comment (sileht) [inline pastamaker/gh_pr.py]: "/" ?

File: pastamaker/gh_pr.py [hunk 2]
```diff
@@ -88,10 +88,17 @@ def pastamaker_travis_post_build_results(self):
...
+                icon = "✅"
+            elif state == "FAILED":
+                icon = "❌"
+            else:
+                icon = ""
+            message.append('- [%s](%s): %s%s' % (
```
Comment (sileht) [inline pastamaker/gh_pr.py]: maybe a space between the "%s" of state and icon?

**Comment by jd** [comment]
Fixed / and added space

**Review by sileht — APPROVED** [review]

**Pull request merged**